renovate.json5

```diff
@@ -1723,15 +1664,15 @@
     {
       "groupName": "common TypeScript types",
       "matchPackageNames": [
-        "@types/node"
+        "@types/node",
```

Comment:
This seems like an unrelated change

Reply:
Yeah I think so too. I rerun our `renovate-config-generator.mjs` script. I moved the unrelated change back manually for now